Makefile

```diff
@@ -5,11 +5,11 @@
 
 sagefile := .sage/bin/sagefile
 
-.PHONY: $(sagefile)
-$(sagefile):
+$(sagefile):.sage/go.mod $(shell find .sage/.. -type f -name '*.go')
 	@cd .sage && go mod tidy && go run .
 
 .PHONY: sage
+sage: clean-sage
```

Comment:
Do we really want to re-download all the tools everytime we rebuild the sagefile binary?

Reply:
No I dont think we should

Reply:
:+1:  Updated such that it is cleaning only the sagefile binary, such that `make sage` forces a recreate.